game_api.txt

```diff
@@ -159,6 +159,16 @@ on_rotate(pos, node, user, mode, new_param2)
 ^ use on_rotate = screwdriver.disallow to always disallow rotation
 ^ use on_rotate = screwdriver.rotate_simple to allow only face rotation
 
+Give Initial Stuff API
+----------------------
+
+give_initial_stuff.give(player)
+^ gives to the player
```

Comment:
what? load time? you do `if minetest.setting_getbool("give_initial_stuff") then`, thats not loadtime for me. Btw. at this point I even agree to you checking the setting every time, and not cacheing it, because its requested only when new players join, which happens seldom enough.